### ISSUE TITLE
How to set weight cost strength in TensorFlow

### ISSUE THREAD

**Issue by btpeter** [tensorflow/tensorflow#876]
When i use Momentum Gradient Descent, how to set weight cost strength?( The λ in this formula.)
![img](http://i.stack.imgur.com/305RE.jpg)

**Comment by vrv** [comment]
This seems like a question better suited for StackOverflow, since this isn't a bug / feature request.
